src/builder.rs

```diff
@@ -621,7 +620,7 @@ impl Builder {
     ///
     /// The returned bundle will have no proof or signatures; these can be applied with
     /// [`Bundle::create_proof`] and [`Bundle::apply_signatures`] respectively.
-    pub fn build<V: TryFrom<i64>>(
+    pub fn build<V: Copy + Into<i64> + TryFrom<i64>>(
```

Comment:
The `Copy + Into<i64>` constraint was added to the `build` function's signature in PR #35 of the `QED-it/orchard` repository:

https://github.com/QED-it/orchard/pull/35

The reason for this, as I understand it, is that in the Zcash repository, the `bvk` value was calculated separately within the `build` function instead of using the `bundle.binding_validating_key()` method (this method requires the `Copy + Into<i64>` constraint).

PR #35 added the burn functionality, making the separate calculation of `bvk` in `build` more complicated. To avoid duplicating code, the calculation was switched to use `bundle.binding_validating_key()`, which required the `Copy + Into<i64>` constraint.

Now, to fix this and remove the `Copy + Into<i64>` constraint while still using `bundle.binding_validating_key()` (and avoiding code duplication), I suggest calculating `native_value_balance` as an `i64` first, then converting it to `V`, which doesn't require `Into<i64>`. Also, to reuse the `binding_validating_key` logic for `bvk`, we could modify it by adding a new free function called `derive_bvk` in the `bundle.rs` module. This function would take three parameters: an `actions` iterator, a generic `value_balance`, and a `burn` iterator. Using iterators would avoid extra memory usage.

We can call `derive_bvk` from `binding_validating_key`, and have `bundle` call `derive_bvk` instead of directly using `binding_validating_key`.

I've implemented this solution in the latest version of the code for this PR.